src/researches/findKeywordInPageTitle.js

```diff
@@ -13,12 +13,28 @@ var escapeRegExp = require( "lodash/escapeRegExp" );
  */
 
 module.exports = function( paper ) {
-	var title = paper.getTitle();
-	var keyword = escapeRegExp( paper.getKeyword() );
-	var locale = paper.getLocale();
-	var result = { matches: 0, position: -1 };
-	result.matches = wordMatch( title, keyword, locale );
-	result.position = title.toLocaleLowerCase().indexOf( keyword );
+	const title = paper.getTitle();
+	// const locale = paper.getLocale();
+
+	const keywordForms = buildKeywordForms( paper );
```

Comment:
This makes sense for this prototype, but for a release version, we need to make sure we don't build the forms separately each time we use them in an assessment. Ideally, the forms should only be built when the keyword is entered the first time or when it's altered.

Reply:
Agreed.